apps/editing-toolkit/editing-toolkit-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -167,33 +166,28 @@ class PageTemplateModal extends Component {
 
 	static getDefaultSelectedTemplate = ( props ) => {
 		const blankTemplate = get( props.templates, [ 0, 'slug' ] );
-		let previouslyChosenTemplate = props._starter_page_template;
+		const previouslyChosenTemplate = props._starter_page_template;
 
-		// Usally the "new page" case
+		// Usually the "new page" case
 		if ( ! props.isFrontPage && ! previouslyChosenTemplate ) {
 			return blankTemplate;
 		}
 
-		// Normalize "home" slug into the current theme.
-		if ( previouslyChosenTemplate === 'home' ) {
-			previouslyChosenTemplate = props.theme;
-		}
-
-		const slug = previouslyChosenTemplate || props.theme;
-
-		if ( find( props.templates, { slug } ) ) {
-			return slug;
-		} else if ( find( props.templates, { slug: DEFAULT_HOMEPAGE_TEMPLATE } ) ) {
-			return DEFAULT_HOMEPAGE_TEMPLATE;
-		}
-		return blankTemplate;
+		// if the page isn't new, select "Current" as the default template
+		return 'current';
```

Comment:
I've got a doubt about this logic here:

```js
if ( ! props.isFrontPage && ! previouslyChosenTemplate ) {
	return blankTemplate;
}
return 'current';
```

As it is written right now, the Current template can only be autoselected when editing the front page.
Is this intentional?
Do we have a reason for checking `previouslyChosenTemplate` if we then end up selecting either Blank or Current (which is a "special" case)?

What about autoselecting Current for _any_ page, as long as they have `previouslyChosenTemplate`?
E.g.
```js
if ( props.isFrontPage || previouslyChosenTemplate ) {
	return 'current';
}
return blankTemplate;
```

It would mean that:
- Front page always autoselect Current. (This is my doubt)
- Other pages autoselect Current if they are using a template, or Blank otherwise (e.g. new pages).

Reply:
I think these two logic expressions are mathematically equivalent via Demorgan?  It seems like they should have the same effect, although the second seems a bit more readable.

We have basically gone from: ` !A && !B => do C `. to ` ! (A || B) => do C` to `A || B => do !C` ? 🤷‍♀️   I think it will be a more readable change but it doesn't seem to cause any difference in functionality?

Reply:
Right, I think I was testing with some very dirty data, and so my change gave me some kind of false positive. 😕 
I'll leave this alone.